react/features/facial-recognition/constants.js

```diff
@@ -6,10 +6,13 @@ export const FACIAL_EXPRESSION_EMOJIS = {
     sad: '🙁',
     surprised: '😮',
     angry: '😠',
-    fearful: '😨',
-    disgusted: '🤢'
+    fearful: '😨'
+
+    // disgusted: '🤢'
```

Comment:
Do we use this facial expression?

Reply:
We do not use it because the disgusted facial expression is not displayed in speaker stats, I followed the design, but if we want to display it, we can do that by uncommenting that line